docs/content/guides/6.multistore/2.tooling-and-concepts/2.file-based-inheritance.md

```diff
@@ -7,6 +7,361 @@ navigation:
+- Choose the store id
+- Choose the parent store - you can decide if the new store should be a child of another store
+- Choose the store type (direct or inheritance-only)
+- Choose the console project name and cloud region
+
+:::info CLI Reference
+For more information on the `storefront-cli` and the commands available, please refer to the [storefront-cli reference](/guides/multistore/tooling-and-concepts/cli-reference).
+:::
+
+:::tip Deployment
+Console project name and cloud region are used when deploying the store. You can read more about it in the [Deployment](/guides/multistore/tooling-and-concepts/deployment/deployment) guide.
+:::
+
+## Managing Store Hierarchy
+
+### Store Structure
+
+Simple hierarchy example:
+```bash
+apps/
+├── storefront-unified-nextjs/     # Base shared code
+└── stores/
+    ├── brand-a/                   # Direct store
+    │   └── storefront-unified-nextjs/
+    └── brand-b/                   # Another direct store
+        └── storefront-unified-nextjs/
+```
+
+Complex hierarchy example with multiple brands and regions:
+```bash
+apps/
+├── storefront-unified-nextjs/          # Base shared code
+└── stores/
+    ├── fashion-brand/                  # Fashion brand stores
+    │   ├── storefront-unified-nextjs/  # Shared fashion brand customizations
+    │   └── stores/
+    │       ├── us-store/              # US fashion store
+    │       └── eu-store/              # EU fashion store
+    └── sports-brand/                   # Sports brand stores
+        ├── storefront-unified-nextjs/  # Shared sports brand customizations
+        └── stores/
+            ├── us-store/              # US sports store
+            └── eu-store/              # EU sports store
+```
+
+#### When to use complex hierarchies
+Complex hierarchies make sense when you have multiple brands or store families that need different:
+- Visual themes
+- Features and functionality
+- Business logic
+- Regional customizations
+
+In the example above, `fashion-brand` and `sports-brand` can have completely different:
+- UI components and styling
+- Customer journey flows
+- Product presentation
+- Regional adaptations
+
+:::warning Keep it Simple
+Only create inheritance levels when you need to share code between multiple stores. If you have just one brand or store family, keep everything in the base shared code (`apps/storefront-unified-nextjs/`).
+:::
+
+### Moving Stores
+
+You can reorganize your store hierarchy using the `store move` command:
+
+```bash
+yarn store move
+```
+
+This command allows you to move stores between different parents and restructure your store hierarchy. For detailed information about managing stores, including moving, renaming, and organizing them, see the [Managing the stores](/guides/multistore/tooling-and-concepts/development/managing-the-stores) guide.
+
+### File Override System
+
+To override a file:
+
+1. Create the same file path in your store's directory
```

Comment:
This sounds too abstract to me. I suggest starting with a concrete example. e.g.:
```
Example: You want to override a header for brand-a store.
1. Copy `apps/storefront-unified-nextjs/components/header.tsx` to `apps/stores/brand-a/storefront-unified-nextjs/components/header.tsx`
2. Modify the file copy to your needs.

Store B will use its own version of `header.tsx`, while inheriting all other files from the base application.

How it works:
During the build, CLI copies the base store and then replaces files with ones from the child stores.
```

Reply:
Updated https://github.com/vuestorefront/vue-storefront/pull/7389/commits/fd0c821d2dbc3ba6982ed075aaec0e5d67ca340a